test/tests/php-fpm-hello-web/run.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -eo pipefail
+
+dir="$(dirname "$(readlink -f "$BASH_SOURCE")")"
+
+image="$1"
+
+# Build a client image with cgi-fcgi for testing
+client_image="php-fpm-client-$RANDOM-$RANDOM"
+docker build -q -t "$client_image" - > /dev/null <<EOF
```

Comment:
I had thought of having some sort of 'post-tests' cleanup script, that would see what tests ran and then clean up their stuff.  This would imply that the first run of the test would set up necessary things, since we do not know the complete list of tests until they are run.

Just my 2 :dollar:/100